source/get-started/containers/container-images.rst

```diff
@@ -0,0 +1,105 @@
+.. _container-images:
+
+|CL-ATTR| container images
+##########################
+
+|CL| can run inside of a container on top of any operating system as long as
+it is hosting a containerized environment, such as Docker* or Kubernetes*.
+Ready-to-use |CL| container images are available on Docker Hub*. 
+
+A |CL| base image is available for standalone use. Additionally, variations of
+popular application images built from the |CL| base image are also published.
+
+All |CL| container images can be `browsed on the Docker Hub* website
+<https://hub.docker.com/search?q=clearlinux&type=image>`_. The corresponding
+Dockerfiles used to build |CL| container images are `on GitHub
+<https://github.com/clearlinux/dockerfiles>`_.
+
+.. contents::
```

Comment:
Remove contents directive.  Doc is too short, and it's not necessary.

Reply:
Agreed. Thank you.